benchmarking/summarize.py

```diff
@@ -201,11 +202,14 @@ def image_quality_vs_time_plot(
     plt.xlim([0.1, 3600])
     plt.semilogx(basex=2)
     plt.xticks(
-        [0.1, 1, 5, 10, 30, 60, 5*60, 10*60, 30*60, 3600,],# 3*3600, 6*3600, 12*3600, 24*3600],
-        ['0.1s', '1s', '5s', '10s', '30s', '1m', '5m', '10m', '30m', '1h', '3h',],# '6h', '12h', '24h'],
+        # 3*3600, 6*3600, 12*3600, 24*3600],
+        [0.1, 1, 5, 10, 30, 60, 5*60, 10*60, 30*60, 3600],
+        ['0.1s', '1s', '5s', '10s', '30s', '1m', '5m', '10m',
+            '30m', '1h']  # , '3h',  # '6h', '12h', '24h'],
```

Comment:
This will probably cause a merge conflict because you also changed this line in the previous PR, but differently. However, I believe you can resolve this conflict appropriately.